src/js/main.js

```diff
@@ -631,4 +632,54 @@ Site.Player = {
   },
 };
 
+Site.EventChecker = {
+  eventUrl: WP.siteUrl + '/wp-json/igv/current-event',
+  eventCheckInterval: 5000,
+  eventData: {},
+  init: function() {
+    var _this = this;
+
+    // Event title element
+    _this.eventTitle = document.getElementById('event-title');
+
+    _this.startChecker();
+  },
+
+  startChecker: function() {
+    var _this = this;
+
+    // Check the current event
+    _this.checkEvent();
+
+    // Start checker on an interval
+    setInterval(function() {
+      _this.checkEvent();
+    }, _this.eventCheckInterval);
+  },
+
+  // Check the current/next event and update background image and show title
+  checkEvent: function() {
+    var _this = this;
+
+    // Make the ajax request
+    $.getJSON(_this.eventUrl, function(data) {
+      if(data) {
```

Comment:
Not only ajax error but a empty return from the API. On that empty return the currently displayed data should be removed.